backend/src/api/auth/auth.routers.js

```diff
@@ -28,4 +30,28 @@ authRouter.post(
   catchAsync(AuthController.loginUser),
 );
 
+authRouter.get(
+  '/google',
+  passport.authenticate('google', { scope: ['email', 'profile'] }),
+);
+
+authRouter.get(
+  '/google/callback',
+
+  passport.authenticate('google', { session: false }),
+  (req, res, next) => {
```

Comment:
First of all, this middleware should be written in controller. Second, what's the reason behind placing another middleware on 52 row? It won't work under any circumstances

Reply:
ok